### ISSUE TITLE
Long alignment didn't work

### ISSUE THREAD

**Issue by baozg** [cmdcolin/jbrowse-plugin-mafviewer#7]
Hi, @cmdcolin 

This plugin looks awesome! I am interested in presenting these variation from whole genome alignment for A.thaliana genomes.
`maf2bed` looks fine, but the jbrowser seems stuck for this long alignment. Do you have any idea about this?

Test on a M2 MacBook with Jbrowser 2.10.3

<img width="903" alt="image" src="https://github.com/cmdcolin/jbrowse-plugin-mafviewer/assets/20680150/7f171f09-996b-4fbb-9744-038d873660b0">
<img width="862" alt="image" src="https://github.com/cmdcolin/jbrowse-plugin-mafviewer/assets/20680150/8151de1e-e06e-40db-a22c-eb9429292ce1">

**Comment by baozg** [comment]
Thanks for your prompt reply. Unfortunately, it throw another error.

<img width="999" alt="image" src="https://github.com/cmdcolin/jbrowse-plugin-mafviewer/assets/20680150/d4b4594d-dda0-4052-b5dc-3d63eedbc0c4">

**Comment by cmdcolin** [comment]
are you hand editing the configuration? Van you paste a copy of the config.json? and do you have the both the maftabix and the tabix index (.tbi)?

**Comment by baozg** [comment]
No, I didn't edit any config. When I also load the `.tbi` into Jbrowser2, the plugin seems to work. But I cannot see my query alignment (11C1). I can send you the genome and bed if you have time to check.

<img width="1006" alt="image" src="https://github.com/cmdcolin/jbrowse-plugin-mafviewer/assets/20680150/1f6fb0af-0341-4910-ad7f-47004927f6d3">

**Comment by cmdcolin** [comment]
that would work great, if you want to send it to me colin.diesh@gmail.com I can check it out

**Comment by cmdcolin** [comment]
This is perhaps late in the game but if you try the pggb or minigraph-cactus pipelines, it can generate some "jbrowse ready" MAF files. it also generates multi-sample VCF which can be viewed 2with the new multi sample vcf view modes https://jbrowse.org/jb2/blog/2025/04/18/v3.3.0-release/

I am also trying to brainstorm more ways to handle stuff like this but hope that helps
